go.mod

```diff
@@ -1,17 +1,14 @@
 module github.com/kubernetes-csi/livenessprobe
 
-go 1.12
+go 1.13
```

Comment:
@pohly had updated CI to use golang 1.15. I'm unsure what it means if this is still on 1.13, but we can figure that out separately.

Reply:
I think it means that the code claims to work with Golang 1.13. It probably still does, but we no longer test that.

Reply:
Sure. I upgraded to 1.15 , so lets keep it in parity with other sidecars :+1:  @msau42 @pohly